ert/data/_record.py

```diff
@@ -400,22 +400,15 @@ async def dump(self, location: Path) -> None:
 
 class InMemoryRecordTransmitter(RecordTransmitter):
     _TYPE: RecordTransmitterType = RecordTransmitterType.in_memory
-    # TODO: these fields should be Union[NumericalRecord, BlobRecord], but that
-    # does not work until https://github.com/cloudpipe/cloudpickle/issues/403
-    # has been released.
-    _data: Optional[record_data] = None
-    _index: Optional[RecordIndex] = None
```

Comment:
How is this not a problem anymore? :thinking: While the fix is merged in cloudpickle, it does not seem to be included in a release, so little surprised that it works

Reply:
Not really sure either, I just wanted to try and see of the fix was merged did not know how to check any other way. Also I was prepared to change the type to `Any` if it was needed :)

Reply:
We have a test that pickles and un-pickles the transmitter and checks that it is fine, so looks like we can ignore this :shrug: